### PR TITLE
Add description label to Schema Registry Dockerfile

### PR DESCRIPTION
**Pull request by amitkgupta**
Some image scanning tools that check for vulnerabilities and OCI image best
practices expect to find a `description` label on images that is not inherited
from base images. The proposed change addresses this requirement.

### CODE REVIEW

**Review by rayokota — APPROVED** [review]
LGTM

**Comment by amitkgupta** [comment]
Hey @rayokota what's the process as it relates to merging PRs for this repo? Should I merge once there's a successful review, or should I wait for one of the owners/maintainers to merge? If I should merge, do you have a preference of `Create a merge commit`, `Squash and merge`, or `Rebase and merge`?

**Pull request merged**
